cassandane/Cassandane/Cyrus/Notify.pm

```diff
@@ -202,7 +202,7 @@ sub test_message
     $status = $talk->get_response_code('status');
     $self->assert_num_equals(2, $status->{'INBOX'}{messages});
     $self->assert_num_equals(3, $status->{'INBOX'}{uidnext});
-
+return;
```

Comment:
what's going on here?

Reply:
Damn.  Debugging cruft.  I will clean that up